alf/algorithms/actor_critic_loss.py

```diff
@@ -48,18 +57,21 @@ def __init__(self,
          - entropy_regularization * entropy)
 
         Args:
-          action_spec: A nest of BoundedTensorSpec representing the actions.
-          gamma: A discount factor for future rewards.
-          td_errors_loss_fn: A function for computing the TD errors loss. This
-            function takes as input the target and the estimated Q values and
-            returns the loss for each element of the batch.
-          use_gae: If True, uses generalized advantage estimation for computing
-            per-timestep advantage. Else, just subtracts value predictions from
-            empirical return.
-          td_lambda: Lambda parameter for TD-lambda computation.
-          entropy_regularization: Coefficient for entropy regularization loss
-            term.
-          td_loss_weight: the weigt for the loss of td error.
+            action_spec (nested BoundedTensorSpec): representing the actions.
+            gamma (float): A discount factor for future rewards.
+            td_errors_loss_fn (Callable): A function for computing the TD errors
+                loss. This function takes as input the target and the estimated
+                Q values and returns the loss for each element of the batch.
+            use_gae (bool): If True, uses generalized advantage estimation for
+                computing per-timestep advantage. Else, just subtracts value
+                predictions from empirical return.
+            use_td_lambda_return (bool): Only effective if use_gae is True.
+                If True, uses td_lambda_return for training value function.
+                (td_lambda_return = gae_advantage + value_predictions)
+            td_lambda (float): Lambda parameter for TD-lambda computation.
```

Comment:
add description for argument normalize_advantages

Reply:
Added